app/Jobs/UpdateWikiMetricDailyJob.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Wiki;
+use App\WikiDailyMetric;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+
+class UpdateWikiMetricDailyJob implements ShouldQueue
+{
+    use Dispatchable;
+    public int $timeout = 3600;
+
+    protected Wiki $wiki;
+
+    public function __construct($wiki)
+    {
+        $this->wiki = $wiki;
+    }
+    /**
+     * Execute the job.
+     */
+    public function handle(): void
+    {
+        $today = now()->format('Y-m-d');
+        $oldRecord = WikiDailyMetric::where('wiki_id', $this->wiki->id)->latest('date')->first();
+        $todayPageCount = $this->wiki->wikiSiteStats()->first()->pages ?? null;
+        $isDeleted = (bool)$this->wiki->deleted_at;
+        if( !$oldRecord || $oldRecord->pages != $todayPageCount || !$oldRecord->is_deleted ) {
+            WikiDailyMetric::create([
+                'id' => $this->wiki->id . '_' . date('Y-m-d'),
+                'pages' => $todayPageCount,
+                'id_deleted' => $isDeleted,
```

Comment:
I discovered trying to run it that this a a typo
```suggestion
                'is_deleted' => $isDeleted,
```